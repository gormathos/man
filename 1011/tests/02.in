sol_id;user_id;task_id;score
128493;1003;2001;6.98
128493;1004;2004;40.54
128501;1002;2001;62.79
128508;1001;2007;0.00
128509;1006;2003;18.42
128518;1002;2004;2.70
128518;1001;2003;26.32
128523;1003;2006;6.67
128524;1003;2007;78.26
128525;1005;2004;32.43
128534;1005;2002;77.78
128534;1006;2007;17.39
128540;1002;2005;83.78
128543;1006;2001;0.00
128544;1003;2004;16.22
128554;1006;2001;60.47
128561;1003;2002;44.44
128564;1004;2004;0.00
128574;1005;2007;0.00
128574;1006;2004;8.11
128574;1003;2004;24.32
128576;1005;2005;35.14
128577;1005;2007;0.00
128579;1001;2002;55.56
128585;1006;2006;22.22
128592;1001;2002;18.52
128601;1006;2007;17.39
128609;1006;2001;11.63
128613;1006;2007;34.78
128622;1005;2004;16.22
128627;1005;2002;11.11
128634;1004;2006;2.22
128638;1001;2003;7.89
128645;1002;2006;91.11
128649;1003;2007;13.04
128651;1006;2007;30.43
128651;1004;2001;11.63
128655;1005;2004;83.78
128660;1005;2004;81.08
128662;1001;2001;0.00
128662;1005;2006;28.89
128668;1006;2006;40.00
128678;1003;2004;24.32
128678;1003;2005;21.62
128679;1006;2003;78.95
128687;1006;2004;16.22
128692;1001;2001;79.07
128697;1005;2006;53.33
128704;1003;2002;18.52
128710;1003;2004;35.14
128711;1003;2006;71.11
128719;1004;2001;30.23
128720;1004;2001;9.30
128728;1004;2003;2.63
128735;1006;2005;67.57
128741;1003;2005;18.92
128742;1001;2007;43.48
128750;1001;2003;15.79
128752;1003;2004;29.73
128753;1002;2001;25.58
128754;1004;2002;55.56
128757;1006;2003;7.89
128757;1006;2003;0.00
128757;1002;2005;37.84
128758;1003;2003;21.05
128761;1004;2001;44.19
128769;1002;2007;60.87
128775;1006;2003;84.21
128778;1001;2001;86.05
128785;1005;2001;34.88
128786;1001;2005;8.11
128792;1005;2001;93.02
128800;1001;2005;35.14
128805;1002;2001;65.12
128810;1005;2002;37.04
128819;1003;2003;2.63
128825;1006;2005;89.19
128827;1001;2001;41.86
128835;1005;2007;52.17
128837;1004;2002;29.63
128840;1005;2006;77.78
128843;1002;2007;34.78
128845;1003;2007;13.04
128851;1006;2004;83.78
128860;1004;2004;10.81
128861;1003;2002;3.70
128871;1003;2001;4.65
128876;1001;2005;24.32
128880;1005;2005;16.22
128884;1005;2001;37.21
128884;1004;2002;3.70
128893;1006;2002;3.70
128893;1002;2003;44.74
128894;1005;2003;73.68
128894;1006;2006;4.44
128896;1005;2003;13.16
128897;1006;2005;16.22
128905;1005;2002;14.81
128905;1002;2006;75.56
128915;1005;2007;0.00
128919;1001;2006;4.44
128920;1003;2005;35.14
128927;1005;2001;55.81
128935;1002;2005;94.59
128945;1001;2006;20.00
128949;1006;2005;29.73
128951;1005;2001;4.65
128958;1001;2002;55.56
128968;1006;2002;66.67
128978;1004;2002;81.48
128985;1005;2001;58.14
128989;1001;2003;15.79
128995;1002;2007;13.04
129001;1001;2004;56.76
129005;1005;2005;16.22
129014;1004;2006;37.78
129019;1002;2003;68.42
129026;1001;2006;33.33
129026;1004;2005;0.00
129030;1002;2003;55.26
129037;1006;2003;65.79
129047;1005;2007;78.26
129048;1003;2002;25.93
129055;1001;2005;32.43
129057;1006;2004;32.43
129057;1001;2002;66.67
129061;1004;2005;0.00
129067;1001;2007;30.43
129068;1002;2003;5.26
129071;1002;2006;0.00
129072;1003;2003;92.11
129074;1002;2002;55.56
129075;1001;2002;66.67
129081;1004;2007;8.70
129082;1006;2007;17.39
129083;1001;2006;35.56
129087;1005;2004;8.11
129090;1003;2007;56.52
129098;1006;2002;40.74
129103;1002;2005;64.86
129111;1003;2006;57.78
129114;1003;2007;26.09
129122;1002;2006;11.11
129127;1001;2007;4.35
129132;1006;2004;43.24
129134;1001;2005;0.00
129134;1001;2004;43.24
129140;1004;2004;24.32
129147;1003;2006;4.44
129147;1005;2003;0.00
129152;1004;2007;26.09
129159;1002;2002;25.93
129161;1006;2004;2.70
129170;1002;2005;8.11
129174;1004;2001;16.28
129179;1004;2005;89.19
129187;1004;2003;63.16
129193;1002;2004;83.78
129201;1005;2003;5.26
129211;1005;2002;70.37
129211;1002;2002;74.07
129216;1004;2006;0.00
129222;1003;2001;86.05
129224;1005;2006;86.67
129231;1002;2004;37.84
129232;1004;2004;21.62
129234;1001;2004;21.62
129242;1006;2002;3.70
129247;1006;2006;4.44
129256;1002;2004;2.70
129258;1006;2006;42.22
129268;1002;2006;40.00
129272;1002;2007;30.43
129273;1006;2006;20.00
129274;1003;2005;10.81
129284;1003;2004;37.84
129292;1001;2004;2.70
129298;1002;2006;64.44
129298;1005;2006;33.33
129298;1002;2003;57.89
129302;1001;2004;67.57
129304;1004;2003;5.26
129309;1002;2002;96.30
129312;1006;2001;90.70
129321;1003;2003;60.53
129327;1004;2004;8.11
129328;1006;2002;44.44
129329;1003;2001;16.28
129335;1001;2002;3.70
129341;1005;2005;0.00
129346;1003;2005;16.22
129356;1002;2001;97.67
129360;1005;2007;100.00
129366;1004;2007;86.96
129374;1004;2003;10.53
129376;1002;2003;63.16
129382;1001;2003;21.05
129391;1003;2005;83.78
129391;1002;2004;62.16
129391;1003;2006;13.33
129397;1004;2007;4.35
129402;1004;2001;2.33
129406;1001;2006;17.78
129410;1004;2007;13.04
129412;1005;2003;31.58
129421;1001;2007;52.17
129427;1002;2007;4.35
129427;1001;2007;0.00
129429;1003;2002;22.22
129429;1005;2002;81.48
129434;1004;2002;96.30
129435;1005;2005;83.78
129438;1006;2001;6.98
129448;1003;2003;97.37
129450;1005;2006;0.00
129460;1004;2006;4.44
129469;1001;2005;81.08
129472;1004;2005;16.22
129482;1001;2003;10.53
129482;1002;2007;13.04
129483;1003;2007;0.00
129485;1004;2005;2.70
129487;1004;2002;100.00
129497;1004;2005;2.70
129501;1003;2001;11.63
129507;1004;2003;10.53
129516;1006;2001;11.63
129519;1005;2004;5.41
129529;1006;2007;0.00
129535;1003;2002;33.33
129544;1006;2002;7.41
129554;1003;2006;15.56
129554;1005;2003;28.95
129555;1004;2007;26.09
129555;1002;2004;18.92
129563;1001;2006;62.22
129570;1005;2005;97.30
129570;1002;2002;11.11
129578;1001;2004;64.86
129580;1003;2001;6.98
129580;1004;2003;34.21
129584;1004;2006;75.56
129588;1001;2001;100.00
129595;1002;2001;76.74
129597;1001;2001;2.33
129601;1003;2003;0.00
129602;1006;2005;54.05
129607;1002;2005;81.08
129609;1002;2002;33.33
129611;1006;2005;0.00
129621;1004;2006;75.56
129627;1002;2001;39.53
