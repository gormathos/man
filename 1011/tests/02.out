1001: Very Good [65.15%]
1002: Excellent [84.68%]
1003: Very Good [71.26%]
1004: Very Good [71.37%]
1005: Excellent [87.99%]
1006: Very Good [70.22%]
